yaas/blueprint.py

```diff
@@ -1,137 +1,54 @@
 # coding: utf-8
 
+""" Interact with Ambari blueprints. """
+
 from __future__ import absolute_import
 from __future__ import print_function
 
-import argparse
-import inspect
 import json
 import pprint
 import requests
 import sys
 
-from . import common
-
-
-def command(parser, args):
-    """ Interact with Ambari blueprints. """
-    subparsers = parser.add_subparsers(dest='subcommand')
-    subcommands = {
-        'add': _add,
-        'list': _list,
-        'remove': _remove,
-        'show': _show,
-        }
-    for name, fn in subcommands.items():
-        subparsers.add_parser(name, help=inspect.getdoc(fn))
-    subargs, extra = parser.parse_known_args(args)
-    subcommands[subargs.subcommand](
-        subparsers.choices[subargs.subcommand],
-        extra)
+from . import config
+from . import utils
 
-
-def _add(parser, args):
+def add(blueprint, blueprint_name, validate_topology=True):
```

Comment:
I think `import`ing will be very magical with this architecture :/

``` python
import yaas
yaas.config.server = 'ambari.example.com`
yaas.blueprint.add(...)
```
...actually it's not too bad! Seems to me a better way would be more like:
``` python
import yaas
server = yaas.Server('http', 'ambari.example.com', 8080)
server.blueprint.add(...)
```
However, that is out of the scope of this PR.